demos/benchmarking/docker-jans-loadtesting-jmeter/yaml/load-test/load_test_dcr.yaml

```diff
@@ -34,7 +34,7 @@ spec:
       - envFrom:
         - configMapRef:
             name: load-test-dcr-cm
-        image: ghcr.io/janssenproject/jans/loadtesting-jmeter:1.0.16_dev
+        image: ghcr.io/janssenproject/jans/loadtesting-jmeter:1.1.4_dev
```

Comment:
Probably need to change tag to 1.1.4-1 instead of `_dev`

Reply:
Oh no actually. Because its in the demo folder i’m actually pushing towards this using the dev image always